### ISSUE TITLE
file missing in folder modules

### ISSUE THREAD

**Issue by dtchantao** [audioku/cross-accent-maml-asr#1]
hi , thanks for your great work. When I try to run your work, error appears:
"ModuleNotFounderror：No module named 'modules.discriminator'

Seems this file is missing.

**Comment by gentaiscool** [comment]
Thanks @dtchantao for informing us. Yes, we forgot to remove that import from the repository. We pushed a change to fix that error.

**Comment by dtchantao** [comment]
> Thanks @dtchantao for informing us. Yes, we forgot to remove that import from the repository. We pushed a change to fix that error.

Hi, gentaiscool, thanks for your reply. Could you please list the version of the package in requirement? Sometimes meet segmentation fault, i think it should be related to the version of packages. Thanks!

**Comment by gentaiscool** [comment]
hi @dtchantao,
Apologies for late reply. Would you please share the error here?

I am using PyTorch 1.3.1, TorchAudio 0.3.1

**Comment by dtchantao** [comment]
hi @gentaiscool

Currently is ok for me. Previous encounter segmentation fault. 
I'm using pytorch 1.4 and torchaudio 0.4.0 now. Seems work for me. Thanks

Another question, does ctc works on your code? It seems ctc needs BLANK token, but you didn't include BLANK token in your vocab.

**Comment by gentaiscool** [comment]
Currently, the CTC training is not tested yet, only the Seq2Seq model works for now.

**Comment by dtchantao** [comment]
when I use CE loss to train on Librispeech. WER seems not dropping.
![image](https://user-images.githubusercontent.com/44595164/90843818-13339000-e395-11ea-914b-d4e48667d78c.png)

**Comment by gentaiscool** [comment]
would you mind sending me the command? And what data did you use for the training?

**Comment by dtchantao** [comment]
hi Thanks, now it works. Another question is that I find the mixed-region setting on your paper: training af , au, ca, en, hk, in, ir, my, nz, sa, sc, sg and us, while in you command you only use af, au, en, hk, in, ir, my, nz, sg and us for training, which one is correct for mixed-region setting? If the latter one is correct, what is the difference between mixed-region setting and cross-region setting? Only the size of the training set? Thanks!

**Comment by SamuelCahyawijaya** [comment]
Hi @dtchantao, the pre-training command can be found in `commands/cv_accent/COMMAND_META.md`, for mixed region command is under the `10 lang` section, while the cross-region setting is under the `us england australia newzealand ireland` section.

To simplify, you can consider regions as continents, which are North America, South America, Europe, Oceania, Asia, Africa, and Antartica.

So basically, in mixed-region setting, we mixed the accents on all regions for pre-training and fine-tuning phase. 
While in the cross-region setting, we pre-train on European, Oceanian, and American accents and fine-tune on Asian and African accents.

I think we need to update the `COMMAND.md` because it is a little bit messy there.
